pkg/checksdb/check.go

```diff
@@ -45,7 +45,8 @@ type Check struct {
 
 	Result         CheckResult
 	CapturedOutput string
-	FailureReason  string
+	Details        string
```

Comment:
If not required outside the `checksdb` package the scope of this field should be restricted to its package. The same applies to the `SkipReason` field.

Reply:
fixed